docs/flowcollector-flows-netobserv-io-v1beta2.adoc

```diff
@@ -217,6 +217,18 @@ This feature requires mounting the kernel debug filesystem, so the eBPF agent po
 It requires using the OVN-Kubernetes network plugin with the Observability feature.
 IMPORTANT: This feature is available as a Developer Preview. +
 
+- `PacketTranslation`: enable enriching flows with packet translation information, such as Service NAT. +
```

Comment:
Commenting here but this comment is for line 218, I think we can remove Developer Preview for this and mention that its now Technology Preview instead.